### PR TITLE
Added FFT and dealt with merge

### PR DESCRIPTION
**Pull request by fuzhanrahmanian**

### CODE REVIEW

File: luna/featurevis/images.py [hunk 1]
```diff
@@ -70,3 +70,138 @@ def save_image(img, name=None):
...
+    return np.sqrt(freq_x * freq_x + freq_y * freq_y)
+
+
+color_correlation_svd_sqrt = np.asarray([[0.26, 0.09, 0.02],
+                                         [0.27, 0.00, -0.05],
+                                         [0.27, -0.09, 0.03]]).astype("float32")
+
+max_norm_svd_sqrt = np.max(np.linalg.norm(color_correlation_svd_sqrt, axis=0))
+
+color_mean = [0.48, 0.46, 0.41]
```
Comment (Sparkier) [inline luna/featurevis/images.py:170]: Are these constants to be used inside of the functions? If so, could you move them to the top and capitalize them?

e.g. COLOR_MEAN

**Review by Sparkier — APPROVED** [review]
Approved, but please consider my comments before merging.

File: luna/featurevis/images.py (hunk 1 above)
Comment (fuzhanrahmanian) [reply luna/featurevis/images.py:170]: Yes they are global variables for now.

File: .gitignore [hunk 2]
```diff
@@ -180,10 +180,11 @@ dmypy.json
...
 
 #output/input files
-.png
-.npy
+*.png
+*.npy
 /output
-
+main.py
+main2.py
```
[the remark this answers is not in the document]
Comment (fuzhanrahmanian) [reply .gitignore:187]: :) I will remove it later xD

**Review by Sparkier — APPROVED** [review]

**Pull request merged**